module-2/backend/week_four.md

```diff
@@ -10,14 +10,49 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+
+stored data from a web site like what's in your shopping cart or links you have clicked on
+
 * What’s the difference between a session and a cookie?
+
+a session in your log in and specific to your user. a cookie is saved data outside of the session
```

Comment:
Sessions allow you to store away individual pieces of information just like with cookies, but the data gets stored on the server instead of the client